src/resources/elements/primeDesignSystem/pselect/pselect.ts

```diff
@@ -44,12 +45,16 @@ export class PSelect {
   @bindable.booleanAttr multiple = false;
   @bindable({defaultBindingMode: bindingMode.twoWay}) value?: string | string[];
   @bindable validationState?: ValidationState;
+  constructor(private readonly taskQueue: TaskQueue){}
 
   refSelectInput: HTMLSelectElement;
   select: SlimSelect;
   isOpen = false;
 
   attached(): void {
+    this.taskQueue.queueTask(() => this.select?.set(this.value));
```

Comment:
i didn't know about taskqueue.  read about it once and forget.  wish I had remembered.